lib/uniffi-zcash/tests/python/orchard.py

```diff
@@ -16,13 +16,20 @@ def test_spending_key_array_mismatch(self):
             ZcashOrchardSpendingKey.from_bytes(key_bytes)
 
     def test_spending_key_from_zip32_seed(self):
-        seed = [1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
-        coin_type = 234
-        account = 2345
+        # seed = [1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+        # coin_type = 234
+        # account = 2345
+        zts = ZcashSpecificTestSupport.from_method("SpendingKey::from_zip32_seed")
```

Comment:
depending on the language, it would be great to put the method name as decorator above the test